mysql/lib/dialect/mysql2_driver_dialect.ts

```diff
@@ -52,4 +54,12 @@ export class MySQL2DriverDialect implements DriverDialect {
   getAwsPoolClient(props: PoolOptions): AwsPoolClient {
     return new AwsMysqlPoolClient(props);
   }
+
+  setupDriverProperties(props: Map<string, any>) {
```

Comment:
Can we use a more specific method name.
This method is setting up network related timeouts.

Reply:
What do you think of `setNetworkTimeouts`